source/content/advanced-global-cdn.md

```diff
@@ -1,75 +1,127 @@
 ---
 title: Advanced Global CDN
-description: Pantheon Professional Services can help you uniquely optimize our Global CDN
+description: Edge configuration tools that enable advanced site management, enhanced security, and a customizable WAF
 categories: [performance]
 tags: [cdn, security, professional-services]
-reviewed: "2020-02-27"
+reviewed: "2020-08-31"
 ---
 
-[Advanced Global CDN](https://pantheon.io/product/advanced-global-cdn) takes our CDN offering a step forward for customers that have unique optimizations required for digital delivery at scale. It includes a wide range of benefits including letting teams individually manage their own sites, drive growth through site iteration, calibrate control over branding, and extend enterprise-grade security.
+[<dfn id="agcdn">Advanced Global CDN</dfn>](https://pantheon.io/product/advanced-global-cdn) is a Managed Service that takes Pantheon's <abbr title="Content delivery network">CDN</abbr> offering a step forward, for customers that require unique optimizations for digital delivery at scale. 
 
-## Frequently Asked Questions
+Each AGCDN service configuration is offered as an annual contract subscription. Customers can include multiple sites to the configuration, or conditionally apply specific rules.
 
-### Is Global CDN still included?
+It includes a wide range of benefits that let teams individually manage their own sites, drive growth through site iteration, calibrate control over branding, and extend enterprise-grade security.
 
-Yes, Global CDN is always included free of charge for all Pantheon customers. This enables customers  to render pages with extraordinary speed by caching responses at the edge at over 70 Fastly points of presence.
+## Features
 
-### Who sets up the Fastly edge configuration files?
+### Full-page & asset caching
```

Comment:
This isn't really accurate for AGCDN - we don't do any additional caching beyond what Global CDN provides.